Cargo.toml

```diff
@@ -18,9 +18,9 @@ backend-sev = ["backend-kvm", "ureq"]
 backend-sgx = ["sgx", "ureq", "x86_64"]
 
 # non-default features
-gdb = ["gdbstub"]
-dbg = []
-disable-sgx-attestation = []
+gdb = ["gdbstub", "enarx-shim-kvm/gdb", "enarx-shim-sgx/dbg"]
```

Comment:
Can this be `gdb = ["dbg", "gdbstub"]`?

Reply:
Wait, is this correct? Why is KVM enabling "gdb" whereas SGX is enabling "dbg"? The names of these flags just need to be changed, they're far too easy to confuse.

Reply:
oops.. typo.. good catch :)

Reply:
yeah, first we had `dbg` ... and then `gdb` was added... feel free to rename either of them and the documentation